hooks/post/30-upgrade.rb

```diff
@@ -38,6 +38,11 @@ def import_subscriptions
   Kafo::Helpers.execute('foreman-rake katello:upgrades:2.4:import_subscriptions')
 end
 
+def remove_elasticsearch
+  Kafo::Helpers.execute('rpm -e elasticsearch lucene3 ruby193-rubygem-tire cigar') if system('rpm -q elasticsearch')
```

Comment:
Perhaps for some, but you can't remove elasticsearch without --nodeps because rubygem-katello requires it? I suppose if the updated RPM is installed first then it might work without it. I was not testing that scenario.